installer-resources/windows/.packaged-lantern.yaml

```diff
@@ -1 +1 @@
-
+startupurl: https://www.facebook.com/manototv
```

Comment:
Oh no, this was commited by error. I actually think we can safely remove those .packaged-lantern.yaml files and add them to the ignore list, as they're generated by the Makefile.